<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_logo_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_historytab">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>0</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border: none;
    padding: 0;
    background: none;
  font-weight: bold;
font-size :15px;
   background-color: transparent; 
    margin: 0 10px;

 }
QPushButton:hover {
 background-color: rgb(37, 153, 255);
 color: white;
 }

</string>
      </property>
      <property name="text">
       <string>Auction history</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_createtab">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border: none;
    padding: 0;
    background: none;
  font-weight: bold;
font-size :15px;
   background-color: transparent; 
    margin: 0 10px;
 }
QPushButton:hover {
 background-color: rgb(37, 153, 255);
 color: white;
 }

</string>
      </property>
      <property name="text">
       <string>Create auction room</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_hometab">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border: none;
    padding: 0;
    background: none;
  font-weight: bold;
font-size :15px;
   background-color: transparent; 
    margin: 0 10px;
 background-color: ;
	background-color: rgba(105, 183, 255, 176);

 }
QPushButton:hover {
 background-color: rgb(37, 153, 255);
 color: white;
 }
QPushButton:focus {
 background-color: rgb(37, 153, 255);
 color: white;
 }

</string>
      </property>
      <property name="text">
       <string>Home </string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>801</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>641</width>
        <height>431</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>639</width>
         <height>429</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>641</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QTextEdit" name="search">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>481</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0.5px solid  rgba(105, 183, 255, 176);
border-radius: 5px;</string>
       </property>
       <property name="placeholderText">
        <string>Search ...</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>353</width>
         <height>26</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Time slot: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="time_from"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="time_to"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="btn_logout">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgba(105, 183, 255, 176);
 border-radius: 5px;
border: 0.5px solid;
 }</string>
      </property>
      <property name="text">
       <string>Log out</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
